src/devices/Pn532/CommandSet.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Pn532
+{
+    /// <summary>
+    /// All cupported command for the PN532
```

Comment:
typo in `supported`

Reply:
also `commands`

Reply:
Yes. I corrected a lot of the grammar and orthography in all the files.